app/main.py

```diff
@@ -1,3 +1,68 @@
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: Union[int, float]) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Union["Distance", int, float]) -> "Distance":
+        return (
+            Distance(self.km + other.km)
+            if isinstance(other, Distance)
+            else Distance(self.km + other)
+        )
```

Comment:
It would be better to place code in multiple lines out of `()` so that if condition would be more readable. Change the code in all related places to encounter that